crates/viewer/re_viewer/src/app.rs

```diff
@@ -1098,6 +1128,40 @@ impl App {
         Some(())
     }
 
+    fn run_copy_time_range_link_command(&mut self, store_context: Option<&StoreContext<'_>>) {
+        let Some(entity_db) = store_context.as_ref().map(|ctx| ctx.recording) else {
+            return;
+        };
+
+        let base_url = match &entity_db.data_source {
+            Some(SmartChannelSource::RerunGrpcStream { url }) => url,
+            // not a recording endpoint
+            _ => return,
```

Comment:
if we can't hide the action in this case, can we at least log a warning that this doesn't have an effect? I fell for this trying to run the command without having a grpc connection and then nothing happened